packages/venia-concept/src/components/Checkout/addressForm.js

```diff
@@ -39,10 +41,29 @@ class AddressForm extends Component {
             postcode: string,
             region_code: string,
             street0: string,
-            telephone: string
+            telephone: string,
+            validation: string
         }),
         submit: func.isRequired,
-        submitting: bool
+        submitting: bool,
+        isAddressIncorrect: bool,
+        incorrectAddressMessage: string
```

Comment:
Maintain alphabetical order on these guys please 👍

Reply:
https://github.com/magento-research/pwa-studio/issues/683. @supernova-at, what do you think about configuring our code quality tools to check this.

Reply:
@supernova-at sorted those ones